viral/grosmark_analysis.py

```
@@ -141,64 +144,75 @@ def get_place_cells(
+    #     / "viral_caches"
+    #     / "place_cells"
+    #     / variable_name
+    #     / f"{session.mouse_name}_{session.date}_rewarded_{rewarded}_{config}_{variable_name}.npy"
+    # )
+
+    get_cache_path = lambda variable_name: (
+        Path("data")
+        / "cache"
+        / variable_name
+        / f"{session.mouse_name}_{session.date}_debugcache_rewarded_{rewarded}_{config}_{variable_name}.npy"
     )
```

Ruff (0.14.1)

Do not assign a `lambda` expression, use a `def`

Rewrite `get_cache_path` as a `def`

(E731)